pipelines/pipeline_katello_nightly.yml

```diff
@@ -11,35 +11,42 @@
   roles:
     - forklift
 
-- hosts: pipeline-katello-nightly-centos7
+- hosts:
+  - pipeline-katello-nightly-centos7
+  - pipeline-proxy-nightly-centos7
   become: yes
   vars:
     katello_repositories_version: nightly
     foreman_repositories_version: nightly
     katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - update_os_packages
+    - foreman_installer
```

Comment:
Yea, thats tough since we install `katello` on the server and `foreman-proxy-content` on the proxy. Technically, `foreman-proxy-content` gets installed on both, so we could install that in parallel and then only install `katello` on the server run?